src/job_titles_feature_extraction.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python
+import re
+
+import nltk
+
+from data_extractor.data_getter import DataGetter
+
+
+# ensure these are downloaded on import.
+nltk.download('wordnet')
+nltk.download('punkt')
+
+
+def pre_process_job_titles():
+    """
+    Process the list of job titles in the CSV training data, doing minor modifications to it,
+    namely, remove cities and stop words.
+    :return: A string with each word separated by a space for the most part. slashes may live.
+    """
+    job_titles = DataGetter.get_raw_training_data("Title")
```

Comment:
Can you pass the feature from main to the function instead of reading it all over again ? We already read the data once in main and pass features to functions to clean things up, titles are already in ` train_raw_data`